handler_success.go

```diff
@@ -219,6 +220,14 @@ func (s *SuccessHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) *http
 		log.Debug("Upstream Path is: ", r.URL.Path)
 	}
 
+	if ctxGetDefaultVersion(r) {
```

Comment:
This code snippet repeated in 3 places, without any changes. I think make sense move it to separate function, and re-use everywhere.